### DISCUSSION TITLE
Including symlink directories in ⌘-P / Go To File dialog

### DISCUSSION THREAD

**Discussion by jordanbrock** [zed-industries/zed#15889] [Feature Requests]
Howdy all,

Is it possible currently, or is it a feature request, to include symlink'd directories in the ⌘-P dialog? For example, at work we have a rails monorepo `/backend` with sub-apps, `/api`, `/scheduler` etc. We share the `app/models` directory across all the sub-apps via a `engines/` directory that is a symlink up the tree to `/backend/shared/engines`

If I do a ⌘-P for one of the model files, nothing appears in the list. Is there a setting I can change to fix this, or should this be a feature request?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
